openbb_terminal/stocks/comparison_analysis/ca_controller.py

```diff
@@ -476,7 +477,15 @@ def call_historical(self, other_args: List[str]):
             type=valid_date,
             default=(datetime.now() - timedelta(days=366)).strftime("%Y-%m-%d"),
             dest="start",
-            help="The starting date (format YYYY-MM-DD) of the stock",
+            help="The starting date (format YYYY-MM-DD) of the stocks",
+        )
+        parser.add_argument(
+            "-e",
+            "--end",
+            type=valid_date,
+            default=(datetime.now() - timedelta(days=366)).strftime("%Y-%m-%d"),
```

Comment:
Suggestion: we can have the end date as today (instead of on it being the same as start date) so we have the `historical` command work out of the box, instead of this:
![image](https://user-images.githubusercontent.com/48914296/210805138-e122d99f-f034-4e53-a7c6-0dc632a03533.png)

Reply:
good call.  silly me copy pasting the start date